### DISCUSSION TITLE
OSI 4계층 , 7계층

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#49] [Q&A] [answered]
OSI 4계층 , 7계층

**Comment by tjdud22** [comment, marked as answer]
TCP/IP 4계층
애플리케이션 계층, 전송 계층, 인터넷 계층, 링크 계층

OSI 7계층
애플리케이션 계층, 프레젠테이션 계층, 세션 계층, 전송계층, 네트워크 계층, 데이터 링크 계층 ,물리계층
